bin/setup.js

```diff
@@ -15,7 +15,15 @@ if (saveRequested) {
   var packageJson = require(path.join(pathToAppRoot, 'package.json'))
 
   packageJson.scripts = packageJson.scripts || {}
-  packageJson.scripts[ 'start' ] = 'hoodie'
+
+  if (packageJson.scripts[ 'start' ]) {
+    log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
+    ', you can start hoodie with "npm run start-hoodie" instead')
+
+    packageJson.scripts[ 'start-hoodie' ] = packageJson.scripts[ 'start' ]
```

Comment:
this should be `packageJson.scripts[ 'start-hoodie' ] = 'hoodie'`, so that we can start the server with `npm run start-hoodie`. The current code sets it to whatever `npm start` does right now, which is not what we want :)